src/header/common/content_length.rs

```diff
@@ -79,7 +79,6 @@ __hyper__tm!(ContentLength, tests {
     test_header!(test1, vec![b"3495"], Some(HeaderField(3495)));
 
     test_header!(test_invalid, vec![b"34v95"], None);
-    test_header!(test_duplicates, vec![b"5", b"5"], Some(HeaderField(5)));
     test_header!(test_duplicates_vary, vec![b"5", b"6", b"5"], None);
```

Comment:
Ok, this change never picks any of the stated values even if they are all identical.